### PR TITLE
add openshift client for openshift v3.6

### PR DESCRIPTION
**Pull request by andreaseger**

### CODE REVIEW

File: 2/Dockerfile [hunk 1]
```diff
@@ -34,7 +34,8 @@ EXPOSE 8080 50000
 RUN curl https://pkg.jenkins.io/redhat-stable/jenkins.repo -o /etc/yum.repos.d/jenkins.repo && \
     rpm --import https://pkg.jenkins.io/redhat-stable/jenkins-ci.org.key && \
     ln -s /usr/lib/jenkins /usr/lib64/jenkins && \
-    INSTALL_PKGS="dejavu-sans-fonts rsync gettext git tar zip unzip openssl bzip2 java-1.8.0-openjdk java-1.8.0-openjdk.i686 java-1.8.0-openjdk-devel java-1.8.0-openjdk-devel.i686 jenkins" && \
+    yum-config-manager --enable rhel-7-server-ose-3.6-rpms && \
+    INSTALL_PKGS="dejavu-sans-fonts rsync gettext git tar zip unzip openssl bzip2 java-1.8.0-openjdk java-1.8.0-openjdk.i686 java-1.8.0-openjdk-devel java-1.8.0-openjdk-devel.i686 atomic-openshift-clients jenkins" && \
```
[the remark this answers is not in the document]
Comment (andreaseger) [reply 2/Dockerfile]: We can readd it although it works fine without it. I already have this branch running for quite a while and been using the openshift client a lot.

**Comment by andreaseger** [comment]
FYI I expect to have a couple more changes to this which we might not be able to do via s2i. But let's see when we get there.

File: 2/Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (nilsding) [reply 2/Dockerfile]: then: fine by me

**Comment by nilsding** [comment]
Will those changes be done within this PR?

**Pull request merged**